From 4cc747f72c446b42da6efe8ea033e69b91ae3870 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 29 Sep 2018 09:56:04 +0800
Subject: [PATCH] fix build failures

- Add `-lpthread' to CFLAGS to fix link error
|undefined reference to symbol 'pthread_create@@GLIBC_2.2.5'

- As HPI-B.01.01/README told us:
  `It is legal to replace it with other SaHpi.h header files,
  as long as the substituted header files is itself compliant.'
  So use the latest openhpi to replace the obsolete one of saftest

Upstream-Status: Pending

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 HPI-B.01.01/Makefile          | 2 +-
 HPI-B.01.01/include/im_spec.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/HPI-B.01.01/Makefile b/HPI-B.01.01/Makefile
index 5d6f0e5..645396b 100644
--- a/HPI-B.01.01/Makefile
+++ b/HPI-B.01.01/Makefile
@@ -35,7 +35,7 @@ build-tests: clean $(BUILD_TESTS:.c=.test)
 tests-pretty:
 	$(MAKE) all | column -t -s:
 
-CFLAGS = -g -W -Wall -D__HPI_B_TEST__ 
+CFLAGS = -g -W -Wall -D__HPI_B_TEST__ -lpthread
 
 INCLUDE = -Iinclude -I../include 
 
diff --git a/HPI-B.01.01/include/im_spec.h b/HPI-B.01.01/include/im_spec.h
index ca54284..11af385 100644
--- a/HPI-B.01.01/include/im_spec.h
+++ b/HPI-B.01.01/include/im_spec.h
@@ -4,4 +4,4 @@
 eg.
  */
 
-	#include <SaHpi.h>
+	#include <openhpi/SaHpi.h>
-- 
2.7.4

